withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -108,10 +108,30 @@ If the request is not successful due to insufficient funds, the poweron should r
 }
 ```
 
+If the request is not successful due to a missing address, the poweron should respond with:
+
+```json
+{
+  "errorCode": 502,
+  "loggingErrorMessage": "TRANPERFORM Error:+TRANERROR"
```

Comment:
Tranperform error?

Reply:
I think that's what Christina originally sent over. I'm going to leave it to you to fill that in with whatever makes sense. We can do something simple like use the strings you mentioned https://github.com/Banno/banno-powerons/pull/8#issuecomment-593438132 or add to them if there's anything else you think would be useful.